apps/node-remote/webpack.config.js

```diff
@@ -6,8 +6,8 @@ const { UniversalFederationPlugin } = require('@module-federation/node');
 
 // Nx plugins for webpack.
 module.exports = composePlugins(withNx(), (config) => {
-  // config.output.publicPath = 'auto'; // this breaks because of import.meta
-  config.output.publicPath = '/'; // this works buy not correct way to do things.
+  config.output.publicPath = 'auto'; // this breaks because of import.meta
```

Comment:
auto now works, public path is calculated in a two-step process. 

1) the hosts who want this remote, know its absolute url to remoteEntry.js 
2) during host initialization, it registers all its known remotes by their globalID and URL. 
3) Host injects remoteEntry or initializes it, when remote starts and needs chunks, it uses webpack_public_path + chunk. 
4) webpack_public_path is calculated by the remote performing a lookup on the registry for its own globalID, where the host would have set its remoteEntry path in order to connect it to webpack. The remote uses this to dynamically find its absolute path, then parse it back into the chunk prefix and patterns set in build.

Since the only location on a server where remote knows the url is the one who asked for the remote to begin. in browser we can use document.currentScript.src to get absolute path, but server has no exact convention unless ESM http imports which wont work.